src/app/modules/item/components/item-children-edit/item-children-edit.component.html

```diff
@@ -46,7 +46,7 @@
 
         <span *ngIf="!data.length">No children for this chapter.</span>
 
-        <alg-add-content [allowSkills]="itemData.item.type === 'Skill'"></alg-add-content>
+      <alg-add-content [allowSkills]="itemData.item.type === 'Skill'" (childrenAdded)="addChild($event)"></alg-add-content>
```

Comment:
the indent deleted in front seems to be an error

Reply:
Fixed